src/main/java/org/primefaces/component/api/UITree.java

```diff
@@ -64,7 +65,9 @@
         requiredMessage,
         skipChildren,
         showUnselectableCheckbox,
-        nodeVar;
+        nodeVar,
+        propagateSelectionDown,
+        propagateSelectionUp;
```

Comment:
If you put these propagate values in UITree shouldn't you remove them from TreeBase.java?

Reply:
I have had this changes and I have correct another mistake I made by setting default propagate values to true.

Reply:
Thanks for fixing.